zipline/pipeline/loaders/blaze/estimates.py

```diff
@@ -82,7 +82,10 @@ def __init__(self,
                  odo_kwargs=None,
                  data_query_time=None,
                  data_query_tz=None,
-                 checkpoints=None):
+                 checkpoints=None,
+                 split_adjustments_loader=None,
```

Comment:
Thoughts on calling this `adjustment_reader`, `adjusted_column_names`, `adjusted_asof`?
Then specifying which adjustments should be applied? e.g. a vendor may also apply dividends to historical data.